envi/symstore/resolver.py

```diff
@@ -324,7 +324,9 @@ def getSymList(self):
         """
         Return a list of the symbols which are contained in this resolver.
         """
-        return [self.getSymByName(name) for name in self.symnames]
+        out = [self.getSymByName(name) for name in self.symobjsbyname]
+        out.extend([self.getSymByName(name) for name in self.symnames])
+        return out
```

Comment:
So one question I have is if all the types in the return list are all the same. Well, that and making sure each element in the list is unique.

Reply:
i'm not sure i understand the question.
currently, the way that PE and ELF parsing are written, one populates symobjsbyname and one populates symnames.

Reply:
except i believe all the tiered-naming is done through symobjsbyname?